spec/integration/runner/variant-reassignment-example-17.spec.js

```diff
@@ -0,0 +1,81 @@
+ * | Product:                           | Product:                                         |                    | Product:                                                     |
+ * | slug: { en: "bike" }               | id: 1                                            |                    | id: 1                                                        |
+ * | masterVariant: { sku: "red-bike" } | slug: { en: "bike" }                             |                    | slug: { en: "bike" }                                         |
+ * | variants: []                       | masterVariant: { sku: "red-tshirt" }             |                    | masterVariant: { sku: "red-bike" }                           |
+ * |                                    | variants: [                                      |                    | variants: []                                                 |
+ * |                                    | { sku: "green-tshirt" },{ sku: "yellow-tshirt" } |                    |                                                              |
+ * |                                    | ]                                                |                    |                                                              |
+ * +------------------------------------+--------------------------------------------------+                    +--------------------------------------------------------------+
+ * |                                    |                                                  |                    | Product:                                                     |
+ * |                                    |                                                  |                    | id: 2                                                        |
+ * |                                    |                                                  |                    | slug: { en: "bike-${timestamp}", ctsd: "${timestamp}" }      |
+ * |                                    |                                                  |                    | masterVariant: { sku: "red-tshirt" }                         |
+ * |                                    |                                                  |                    | variants: [{ sku: "green-tshirt" },{ sku: "yellow-tshirt" }] |
+ * +------------------------------------+--------------------------------------------------+--------------------+--------------------------------------------------------------+
+ */
+/* eslint-enable max-len */
+describe('Variant reassignment', () => {
+  let ctpClient
+  let product1
+  const logger = utils.createLogger(__filename)
+  const product1Sku1 = 'red-tshirt'
+  const product1Sku2 = 'green-tshirt'
+  const product1Sku3 = 'blue-tshirt'
+  const product1Slug = { en: 'bike' }
+
+  before(async () => {
+    ctpClient = await utils.createClient()
+    const products = await utils.createCtpProducts([[product1Sku1, product1Sku2, product1Sku3]],
```

Comment:
maybe it will be much easier if you have one array `productSkus` and use it across whole test

Reply:
Can you explain a bit more what you mean?

Reply:
I don't think it's easier because I use the array with 3 elements just here. At the other places, I use array of 1 element or just the SKU element alone.